src/sentry/suspect_resolutions/resolved_in_active_release.py

```diff
@@ -0,0 +1,38 @@
+from datetime import timedelta
+
+from django.utils import timezone
+
+from sentry.models import Deploy, Group, GroupStatus, Release
+
+
+def is_resolved_issue_within_active_release(issue_id: int) -> bool:
+    # check that the issue is resolved and is part of a release
+
+    issue = Group.objects.filter(id=issue_id).first()
+
+    if issue is None or issue.status != GroupStatus.RESOLVED or issue.get_last_release() is None:
+        return False
+
+    # returns the release version associated with the issue
+    latest_release_version_issue = issue.get_last_release()
+
+    # get release object from version
+    latest_release_issue = Release.objects.filter(version=latest_release_version_issue)
+
+    # get latest deploy of the release associated with the issue
+    latest_deploy_release: Deploy = (
+        Deploy.objects.filter(release_id=latest_release_issue.first().id)
```

Comment:
This `.first().id` call could result in a `NoneType` error here if `last_release_issue` returns nothing.